tools/ctl/ctl.go

```diff
@@ -252,11 +252,17 @@ var runBenchmark = &cobra.Command{
 		if err != nil {
 			log.Fatal(errors.Wrap(err, "parsing API endpoint"))
 		}
-		apiURL.Scheme = "https"
 		ctx := cmd.Context()
-		idclient, err := oauth.AuthorizedUserIDClient(ctx)
-		if err != nil {
-			log.Fatal(errors.Wrap(err, "creating authorized HTTP client"))
+		var idclient *http.Client
+		if strings.Contains(apiURL.Host, "run.app") {
```

Comment:
HasSuffix? and that should probably be `.run.app`

Reply:
Resolved in https://github.com/google/oss-rebuild/pull/35